### DISCUSSION TITLE
Rest + camunda

### DISCUSSION THREAD

**Discussion by wkharrat** [flowsforapex/apex-flowsforapex#363] [General]
Thanks for this great product. Compared to using a calls of rest and camunda process, is this any limitation if i use your product? Execuse my question, but i found your product in time when we decide to use camunda with apex.

**Comment by LouisMoreaux** [comment]
Hi,

First thanks, we love to see feedbacks like that! 

What kind of limitations are you thinking about?

We do not support the full BPMN specification right now. I recommend to take a look at our documentation to see if it fits your needs ([https://flowsforapex.org/latest/bpmn/](https://flowsforapex.org/latest/bpmn/))

**Reply by wkharrat** [reply]
thank you for this encouraging feedback, i just begin to consider to expose that to our clients, we have many clients developing with apex and we support them in some fonctionnality, so the workflow need exist and i begin to have idea about this. Consider if i put workflows applications in a workspace (ask for holiday, expense workflow...) i think there's apex limitation to use them in application in another workspace, example i can add icon in general ledger application it will be active when i have tasks to do from workflow application. I can do that with ords service based on views query... i can add some ords service to visualize the items( like the list/form to approve expense in your tutorial) but to visualize the current state of the workflow (using the bpmn.io) is hard if there's no pl/sql api to do that. I know long replay. But i can't ask user to logout from general ledger and go to workflow application if it have new tasks to do. I know these is apex limitation but we talk about transversal applications. Even if there's no direct solution, but passing about ords services to manipulate workflows is a real advantage for my clients

**Reply by LouisMoreaux** [reply]
Hello,

The Viewer plug-in works with different kind of data sources so you can create an ORDS RESTful Service to expose the content of the flow_instance_details_vw view for a specific prcs_id, create a REST data source in your APEX application on top of the ORDS service and then change the location attribute of the Viewer plug-in to use the REST service instead of the local database.

![image](https://user-images.githubusercontent.com/42840524/142003994-018f1849-2f9d-42c4-bf3c-7dadae753594.png)

**Comment by nielsdebr** [comment]
Can we talk about your case in a Zoom session? Please DM me on Twitter. My handle is @nielsdb. Looking forward to exchanging some thoughts.
